README.md

```diff
@@ -355,6 +355,9 @@ interface Tweet extends Node {
 interface FlourishReference extends Reference {
 	referencedType: "flourish"
 	flourishType: string
+	layoutWidth: "" | "full-grid"
```

Comment:
Interestingly in our content pipeline API we transform this into a boolean (`fullGrid: true`). Should we do the same here? Or are we trying to be consistent with other things that have this? (Layouts?)

Reply:
Layout values:
`full-grid`, `fullGrid`, `inset-left`, `true`.

I'm pretty sure `full-grid`/`fullGrid` means different things for all instances.

Reply:
Okay, maybe:
- `""` = normal width (the width of the article copy)
- `inset-left` = inset left (for things that allow it)
- `full-grid` = bleeds to the full width of the grid 

I think that works and makes sense. So I think: 
- keep this as is
- CardLayout when we do it should follow the same pattern.
- Our CP content pipeline API can keep it as a string (and change it to boolean if it needs to in the JSX)


Gosh that was a journey.

Reply:
i probably should have written this down, because i went on this same journey before writing that line in the first place. 🙈